aws/resource_aws_ecs_service_test.go

```diff
@@ -148,13 +148,30 @@ func TestAccAWSEcsService_basicImport(t *testing.T) {
 				ImportState:       true,
 				ImportStateVerify: true,
 			},
-			// Test non-existent resource import
+		},
+	})
+}
+
+func TestAccAWSEcsService_disappears(t *testing.T) {
+	var service ecs.Service
+	rString := acctest.RandString(8)
+
+	clusterName := fmt.Sprintf("tf-acc-cluster-svc-w-arn-%s", rString)
+	tdName := fmt.Sprintf("tf-acc-td-svc-w-arn-%s", rString)
+	svcName := fmt.Sprintf("tf-acc-svc-w-arn-%s", rString)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSEcsServiceDestroy,
+		Steps: []resource.TestStep{
 			{
-				ResourceName:      resourceName,
-				ImportStateId:     fmt.Sprintf("%s/nonexistent", clusterName),
-				ImportState:       true,
-				ImportStateVerify: false,
-				ExpectError:       regexp.MustCompile(`No ECS service found`),
```

Comment:
We actually want to keep this test as its currently the only way we can verify the condition you were originally seeing. We just needed to grab something out of the new error message from import failing (properly!):

```
--- FAIL: TestAccAWSEcsService_basicImport (43.46s)
    testing.go:520: Step 2, expected error:

        1 error occurred:
        	* aws_ecs_service.jenkins (import id: tf-acc-cluster-svc-ye9z6h7j/nonexistent): 1 error occurred:
        	* import aws_ecs_service.jenkins result: nonexistent: import aws_ecs_service.jenkins (id: nonexistent): Terraform detected a resource with this ID doesn't
        exist. Please verify the ID is correct. You cannot import non-existent
        resources using Terraform import.





        To match:

        No ECS service found
```

I went with:

```
ExpectError:       regexp.MustCompile(`Please verify the ID is correct`),
```